### PR TITLE
feat(judicial-system): Get police case file for original ancestor

### PR DESCRIPTION
**Pull request by gudjong**
# Get police case file for original ancestor

https://app.asana.com/0/1199153462262248/1201185512430557

## What

☝️ 

## Why

Required

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/modules/case/interceptors/test/caseOriginalAncestorInterceptor.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  describe('next call', () => {
+    const theCase = {} as Case
+    const request = { case: theCase }
+    const result = {}
+    let then: Then
+
+    beforeEach(async () => {
+      mockRequest.mockImplementationOnce(() => request)
+      // mockFindOriginalAncestor.mockResolvedValueOnce(theCase)
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/case/interceptors/test/caseOriginalAncestorInterceptor.spec.ts:67]: Consider removing this unused code

File: apps/judicial-system/backend/src/app/modules/case/guards/caseCompleted.guard.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { Case } from '../models'
 
 @Injectable()
 export class CaseCompletedGuard implements CanActivate {
-  async canActivate(context: ExecutionContext): Promise<boolean> {
+  canActivate(context: ExecutionContext): boolean {
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/case/guards/caseCompleted.guard.ts:15]: I see that you are removing async in a couple of places but you are not removing any awaits. Was this function never being awaited on?

**Review by oddsson — CHANGES_REQUESTED** [review]
Some minor things

File: apps/judicial-system/backend/src/app/modules/case/guards/caseCompleted.guard.ts (hunk 2 above)
Comment (gudjong) [reply apps/judicial-system/backend/src/app/modules/case/guards/caseCompleted.guard.ts:15]: This function is called as part of middleware execution. The framework uses awaits as neede.

**Review by oddsson — APPROVED** [review]

**Pull request merged**